sdk/maps/azure-maps-service/samples/common/common.py

```diff
@@ -0,0 +1,50 @@
+from azure.maps.service import MapsClient
```

Comment:
We should not have a common helper file for our samples. This a) makes the samples much harder to follow and b) indicates that our client library is missing functionality. We do not want users of our library to have to write boilerplate code in order to use it.